dev/CommonStyles/FlipView_themeresources.xaml

```diff
@@ -109,15 +124,15 @@
                         BorderBrush="{TemplateBinding BorderBrush}"
                         BorderThickness="{TemplateBinding BorderThickness}"
                         contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
-                        contract7NotPresent:CornerRadius="{ThemeResource ControlCornerRadius}">
+                        contract7NotPresent:CornerRadius="{TemplateBinding CornerRadius}">
```

Comment:
This won't work, on rs4 and below the control doesn't have a corner radius to bind to.  Please go back to using the theme resource downlevel.

Reply:
Reverted all changes to this property